core/src/ibc/channel_04/manager.rs

```diff
@@ -54,12 +54,17 @@ impl<'a> Manager<'a> {
     }
```

Comment:
What is "Acked-by: Park Juhyung <jh@codechain.io>" in your commit message? ('Fix connection hop to use the same chain's identifier')

Reply:
I added it by mistake. I'll remove it.